distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Text_Sub_Range.enso

```diff
@@ -40,6 +40,15 @@ type Text_Sub_Range
        Select an empty string if the input does not contain `delimiter`.
     After_Last (delimiter : Text)
 
+    ## PRIVATE
+       Convert to a display representation of this Index_Sub_Range.
+    to_display_text : Text
+    to_display_text self = case self of
```

Comment:
There shall be generic implementation for these types with few atoms that would fit the default needs. At least I see some pattern that could be handled in a generic way via `Meta` or in the engine itself.